assets/src/application/ui/layout/confirm/index.ts

```diff
@@ -1 +1,2 @@
+export { default as ConfirmClose } from './ConfirmClose';
 export { default as ConfirmDelete } from './ConfirmDelete';
```

Comment:
hmmmm... dunno about these being in `/ui/layout/` as they are not really structural (ie: control layout/positioning of other components)

maybe move these into `/ui/input/` since they essentially require user input

Reply:
Moved, thx.